### DISCUSSION TITLE
Self Hosted Runner, is the version released by GitHub available for GitHub Enterprise Server 3.8.2?

### DISCUSSION THREAD

**Discussion by kmeratia** [community/community#61471] [Enterprise] [answered]
### Select Topic Area

Question

### Body

Is the version release by GitHub with [gha-runner-scale-set](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/quickstart-for-actions-runner-controller). Supported by GitHub Enterprise Server 3.8.2. 

There is a comment on https://github.com/actions/actions-runner-controller/discussions/2174#discussioncomment-5769497. From @Link- that "Support for GHES will only come in version 3.9"

I am getting an error while installing gha-runner-scale-set. 
"error": "404 - had issue communicating with Actions backend: Request returned status: 404 Not Found"

**Comment by Link-** [comment, marked as answer]
runner scale sets are only availabee in GHES 3.9 and above.

![image](https://github.com/community/community/assets/568794/e293dae1-f53b-459d-b8d1-a4ddd9fe08df)
